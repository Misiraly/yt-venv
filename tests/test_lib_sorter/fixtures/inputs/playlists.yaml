rock:
- f65df1d34a8a0d13
- d2b5e25648866d79
- e8867ad225416a72
- 6c3b53d15740656a
