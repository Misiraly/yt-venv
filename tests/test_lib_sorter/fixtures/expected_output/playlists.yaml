rock:
- f65df1d34a8a0d13
- d2b5e25648866d79
- e8867ad225416a72
- 82141a4385f8c2c2
- 91c40bf78a18c2c5
funky:
- 82141a4385f8c2c2
- 495dfa5d930cb050
- 91c40bf78a18c2c5
- 6ce30a3874668de7
- 6ce30a3874668de7